person.rb

```diff
@@ -2,14 +2,15 @@
 
 class Person < Nameable
   attr_accessor :name, :age, :parent_permission
-  attr_reader :id
+  attr_reader :id, :rentals
 
   def initialize(name: 'Unknown', age: 0, parent_permission: true)
     super()
     @id = generate_id
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
   end
 
```

Comment:
- [ ] It would be nice if you have a method to create rentals to form the many-to-many relationship 👍 the method should look like below ⬇️

```ruby
  def add_rental(book, date)
    # *The person itself should be sent as a parameter to create the new rental achieving the 'has-many' association
    Rental.new(date, book, self)
  end
```